libvirt/network_def.go

```diff
@@ -57,3 +59,43 @@ func newNetworkDef() libvirtxml.Network {
 		return d
 	}
 }
+
+func getHostXMLDesc(ip, mac, name string) string {
```

Comment:
this function will be usefull when we will update the network ( see current limitations)